onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -2787,7 +2839,15 @@ def test_manager_can_update_xform_xls_file(self):
             self.assertEqual(response.status_code, 403)
 
         # assign manager role
-        ManagerRole.add(self.user, self.xform.project)
+        data = {
+            "project": self.xform.project.id,
+            "username": self.user.username,
+            "role": "manager",
+        }
+
+        serializer = ShareProjectSerializer(data=data)
+        if serializer.is_valid():
```

Comment:
We can omit serializer.is_valid() since for tests it doesn't matter if an exception is thrown

Reply:
```
AssertionError: You must call `.is_valid()` before calling `.save()`.
```